src/containers/SignUp/index.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable prettier/prettier */
+import React from 'react';
+import {View, Text, SafeAreaView} from 'react-native';
+import {Formik} from 'formik';
+import { CheckBoxField, TextInputField, CustomButton } from '../../components';
+
+const SignUp = () => {
+    return (
+        <SafeAreaView>
+            <Formik>
+            <View>
+               <View>
+               <TextInputField label='Nombre'/>
+               </View>
+                <View>
+                   <TextInputField label='Correo'/>
+                </View>
+                <View>
+                   <TextInputField label='Contraseña'/>
+                </View>
+                <View>
+                   <CheckBoxField label='Acepto los Terminos y Condiciones.'/>
+                </View>
+                <View>
+                   <CustomButton text='Registrarme'/>
+                </View>
+                <View>
+                   <CustomButton text='Registrarme con Google'/>
```

Comment:
Codacy has a fix for the issue: [Unexpected usage of singlequote.](https://app.codacy.com/gh/BC-MAY-21-RN/proyecto-proy-t01/pullRequest?prid=7869790)
```suggestion
                   <CustomButton text="Registrarme con Google"/>
```